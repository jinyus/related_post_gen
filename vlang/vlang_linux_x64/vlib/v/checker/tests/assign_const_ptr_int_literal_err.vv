const a = 3

fn main() {
	b := &a
	dump(b)
}
