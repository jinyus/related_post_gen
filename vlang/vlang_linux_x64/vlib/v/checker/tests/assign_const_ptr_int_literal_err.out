vlib/v/checker/tests/assign_const_ptr_int_literal_err.vv:4:8: error: cannot assign a pointer to a constant with an integer literal value
    2 | 
    3 | fn main() {
    4 |     b := &a
      |           ^
    5 |     dump(b)
    6 | }
Details: Specify the type for the constant value. Example:
         `const a = int(3)`